core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.js

```diff
@@ -30,10 +30,10 @@ oppia.directive('audioTranslationBar', [
 
         $('.oppia-translation-tab').on('dragover', function(evt) {
           evt.preventDefault();
-          if (!scope.showDropArea) {
-            scope.showDropArea = true;
-            scope.$digest();
-          }
+          // don't show for user that is logged in and cannot add translation.
```

Comment:
Start this comment with a capital letter! Also, you can explain what "don't show".

Reply:
ok changed it to "Don't show drag and drop area for user that is logged in and cannot add translation." So should be done.